README.md

```diff
@@ -86,6 +78,15 @@ This image can be run with a read-only container filesystem. For details please
 
 This image can be run with a non-root user. For details please [read the docs](https://docs.linuxserver.io/misc/non-root/).
 
+### Media folders
+
+We have set /music and /downloads as optional paths, this is because it is the easiest way to get started. While easy to use, it has some drawbacks. Mainly losing the ability to hardlink (TL;DR a way for a file to exist in multiple places on the same file system while only consuming one file worth of space), or atomic move (TL;DR instant file moves, rather than copy+delete) files while processing content.
```

Comment:
why did we change from /tv to /music for sonarr?

Reply:
We don't, we just need to wait on the mentioned monthly